composer.json

```diff
@@ -21,8 +21,9 @@
     ],
     "require": {
         "php": "^8.1",
-        "nesbot/carbon": "^2.72.1",
-        "ext-calendar": "*"
+        "ext-calendar": "*",
+        "geniusts/hijri-dates": "^1.1",
```

Comment:
Not using this library for a couple of reasons:
1. https://github.com/spatie/holidays/pull/155#issuecomment-1904467312
2. https://github.com/spatie/holidays/blob/main/src/Countries/Turkey.php#L11